.github/workflows/deploy-stg-enhanced-worker.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  push:
+    paths: ["src/**", "cdk-infra/lib/constructs/worker/**"]
```

Comment:
Should we add `Dockerfile.enhanced` or is that unnecessary? Would we also need to consider updates to anything in the `modules/` directory or package files?

Reply:
Yes to both! I missed the `Dockerfile.enhanced` and I hadn't thought about modules. Will update.